common/tmx_level.go

```diff
@@ -72,24 +72,24 @@ type TMXData struct {
 	// valid values are one of "", "zlib", "gzip"
 	Compression string `xml:"compression,attr"`
 	// Data contains an encoded list of uint32 guids defining
-	// the tile layout of a layer
+	// the Tile layout of a layer
 	Data string `xml:",innerxml,"`
 	// Tiles is an array of tiles containing guids. Not set if
 	// other encodings are used
-	Tiles []TMXTile `xml:tile`
```

Comment:
Don't change the xml parsing comment to be capitalized, that may mess up unmarshalling

Reply:
Ah that was Goland's super renaming refactoring functionality. Sorry, one has to do everything manually.

Reply:
No worries, I only knew to be wary of that because I've done exactly that myself before lol